hippod/snippet_db.py

```diff
@@ -65,14 +78,13 @@ def execute_snippet(app, sha, data, source_path, source_type):
     # os.system('python3')
     # for lib in lib_list:
     #   os.system('apt-get install {}'.format(lib))
-    print(snippet_db_path)
 
     s_type = mime_type.split('-')[2]
     if s_type == 'python' or s_type == 'python2' or s_type == 'python3':
         snippet_tmp_path = os.path.join('/tmp', 'tmp{}.py'.format(sha))
     exec_code = os.system('python3 {} {}'.format(snippet_tmp_path, snippet_db_path))
     if exec_code == 0:
-        exchange_entry(app, sha, data, source_path, source_type)
+        exchange_entry(app, sha, data, source_path, source_type, snippet_db_path)
```

Comment:
exchange_entry is a shitty name? What is exchanged? If this is the last idea that we need a comment what happens and what are you trying to solve here. More comments ...